tox.ini

```diff
@@ -13,6 +13,15 @@ commands =
     coverage erase
     py.test {posargs}
 
+[testenv:docs]
+basepython = python3.6
+setenv =
+    PYTHONHASHSEED = 0
+deps = -rdocs-requirements.txt
+commands =
+    sphinx-build -W -b html -d {envtmpdir}/doctrees docs docs/_build/html
```

Comment:
This [builds the docs](http://www.sphinx-doc.org/en/stable/man/sphinx-build.html), but the `-W` turns warnings into errors. Errors could be having an `foo.rst` file within the `docs/` dir, but it not included/linked anywhere, or bad [directives](http://www.sphinx-doc.org/en/stable/rest.html#directives), bad links, many things.